lib/wovnrb/url_language_switcher.rb

```diff
@@ -26,6 +26,30 @@ def add_lang_code(href, to_lang_code, headers)
       end
     end
 
+    # Removes language code from a URI component (path, hostname, query etc), and returns it.
+    #
+    # @param uri  [String] original URI component
+    # @param lang [String] language code
+    # @return     [String] removed URI
+    def remove_lang_from_uri_component(uri, lang)
+      lang_code = @store.settings['custom_lang_aliases'][lang] || lang
+
+      return uri if lang_code.nil? || lang_code.empty?
```

Comment:
`lang_code.blank?` can be used instead.

Reply:
Good find. I was always hesitant to use `blank?` because it's provided by Rails. But we can use it since wovnrb included `ActiveSupport`.